### DISCUSSION TITLE
Welcome to shadowsocks-windows Discussions!

### DISCUSSION THREAD

**Discussion by database64128** [shadowsocks/shadowsocks-windows#3041] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by k1nd2020** [comment]
start

**Comment by xiangrikui88200** [comment]
请问版主：今天已将电脑的ss客户端升级至4.4.0.185最新版本，加密方式已不支持aes-256-cfb了，那其他网站提供的ss节点是这个加密方式，我该如何添加呀？刚才试了，扫描QR二维码后提示需转换为chacha20-ietf-poly1305的加密方式，我按照提示同意添加后，却无法联网。请问该如何解决？谢谢，期待您的回复解答！

**Reply by chenshaoju** [reply]
个人建议联系你的服务商，推动升级。

如果服务商不愿意升级，旧版本仍然可用（记得关闭自动升级）：https://github.com/shadowsocks/shadowsocks-windows/releases/tag/4.3.3.0

**Reply by xiangrikui88200** [reply]
哦，明白了，谢谢您的解答！

**Comment by zyhhh2525** [comment]
想问一下ss怎么用啊，在哪可以买？

**Comment by obmeotoff** [comment]
Hi people! when enabled SS stop working Xbox App and Microsoft Store! who knows how to solve the problem?
p.s. Windows 11
